_posts/2024-09-20-open-mic-generating-documentation.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: "Upcoming Open Mic - Generating and Publishing Ontology Documentation"
+categories: [Open Mic Session, Open Mic]
+---
+The Open mic session took place on Friday 20th June 2024. The speaker [Michal Med](https://kbss.felk.cvut.cz/web/team#michal-med) introduced his research in the automatic generation and publication of documentation of ontologies. This session follows up the speech from December 2023[1]. The whole speech, including the discussion, was as usual recorded. The video is available here:
+
+{% include video id="Pu-T6jjOrVE" provider="youtube" %}
+
+##### Problem?
+The speech is trying to get the solution to the following problem: **how to automatically update the ontology documentation on the web when the ontology is updated?**
+
+{% include figure image_path="/assets/images/posts/2024-09-20-problem.png" alt="Problem definition." %}{: .post-image }
+
+There are plenty of options on how to do this, but most of them have to solve following four questions:
+1) Where to store the ontology/ies?
+2) How to find out that the ontology changed?
+3) How to (re)generate the documentation with as much variables recieved from the ontology?
```

Comment:
```suggestion
3) How to (re)generate the documentation with many variables recieved from the ontology?
```

Reply:
Not sure if this is what you meant, but it was strange for me to read "as much" without context.

Reply:
tam bylo myšleno "as much ... as possible"